tests/test_linkml_files.py

```diff
@@ -1,64 +1,196 @@
-    """ Test that linkml_model/linkml_files.py work """
-    def test_basic_rules(self):
-        self.assertEqual("https://w3id.org/linkml/annotations.yaml",
-                         URL_FOR(Source.ANNOTATIONS, Format.YAML))
-        self.assertEqual("https://w3id.org/linkml/meta.model.context.jsonld",
-                         URL_FOR(Source.META, Format.NATIVE_JSONLD))
-        self.assertEqual(os.path.join(root_path, "model/schema/meta.yaml"),
-                         LOCAL_PATH_FOR(Source.META, Format.YAML))
-        print(LOCAL_PATH_FOR(Source.META, Format.YAML))
-        self.assertEqual(os.path.join(root_path, "jsonld/types.model.context.jsonld"),
-                         LOCAL_PATH_FOR(Source.TYPES, Format.NATIVE_JSONLD))
-        self.assertEqual("https://linkml.github.io/linkml-model/model/schema/meta.yaml",
-                         GITHUB_IO_PATH_FOR(Source.META, Format.YAML))
-        self.assertEqual("https://linkml.github.io/linkml-model/jsonld/types.model.context.jsonld",
-                         GITHUB_IO_PATH_FOR(Source.TYPES, Format.NATIVE_JSONLD))
-        self.assertEqual("https://raw.githubusercontent.com/linkml/linkml-model/main/jsonld/meta.model.context.jsonld",
-                         GITHUB_PATH_FOR(Source.META, Format.NATIVE_JSONLD, ReleaseTag.LATEST))
-        self.assertEqual("https://raw.githubusercontent.com/linkml/linkml-model/testing_branch/owl/mappings.owl.ttl",
-                         GITHUB_PATH_FOR(Source.MAPPINGS, Format.OWL, branch="testing_branch"))
-
-    @unittest.skipIf(SKIP_GITHUB_API, "Github API tests skipped")
-    def test_github_specific_rules(self):
-        """
-        Test accesses that require github API to access
+
+def test_basic_rules():
+    assert "https://w3id.org/linkml/annotations.yaml" == URL_FOR(Source.ANNOTATIONS, Format.YAML)
+    assert "https://w3id.org/linkml/meta.context.jsonld" == URL_FOR(Source.META, Format.NATIVE_JSONLD)
+    assert os.path.join(root_path, "model/schema/meta.yaml") == LOCAL_PATH_FOR(Source.META, Format.YAML)
+    print(LOCAL_PATH_FOR(Source.META, Format.YAML))
+    assert os.path.join(root_path, "jsonld/types.context.jsonld") == LOCAL_PATH_FOR(Source.TYPES, Format.NATIVE_JSONLD)
+    assert "https://linkml.github.io/linkml-model/latest/linkml_model/model/schema/meta.yaml" == GITHUB_IO_PATH_FOR(
+        Source.META, Format.YAML)
+    assert "https://linkml.github.io/linkml-model/latest/linkml_model/jsonld/types.context.jsonld" == \
+           GITHUB_IO_PATH_FOR(Source.TYPES, Format.NATIVE_JSONLD)
+    assert "https://raw.githubusercontent.com/linkml/linkml-model/main/linkml_model/jsonld/meta.context.jsonld" == \
+           GITHUB_PATH_FOR(Source.META, Format.NATIVE_JSONLD, ReleaseTag.LATEST)
+    assert "https://raw.githubusercontent.com/linkml/linkml-model/testing_branch/linkml_model/owl/mappings.owl.ttl" == \
+           GITHUB_PATH_FOR(Source.MAPPINGS, Format.OWL, branch="testing_branch")
+
+
+SKIP_GITHUB_API = False
```

Comment:
ideally this would be a pytest option or dependent on some flag, if we want to keep skipping them - i marked them as skip because they kept failing bc the ratelimits on github API were very low, but if we want to rescue them then yes a flag that can be accessed would be good :)